mindmeld/components/question_answerer.py

```diff
@@ -797,13 +826,22 @@ def build_query(self):
                         }
                     }
                 else:
-                    clause = {
-                        "match": {
-                            self.field + ".normalized_keyword": {
-                                "query": self.value
+                    if self.query_type == 'text':
+                        clause = {
+                            "match": {
+                                self.field + ".char_ngram": {
```

Comment:
just wonder is this supposed to be against `processed_text` rather than `char_ngram` field?

Reply:
For filters, the default system uses the normalized words. if the query_type is text, we want to incorporate some fuzziness in the filter as well to give the user flexibility.